redshift_sqlalchemy/dialect.py

```diff
@@ -189,17 +279,106 @@ def set_isolation_level(self, connection, level):
 
         connection.set_isolation_level(level)
 
-    def _get_column_info(self, *args, **kwargs):
+    def _get_column_info(self, col, domains):
         column_info = super(RedshiftDialect, self)._get_column_info(
-            *args,
-            **kwargs
-        )
+            col.name, col.format_type, col.default, col.notnull,
+            domains, [], col.schema)
```

Comment:
for really long argument lists like this, I like to specify the keyword arguments, so that it's hard to accidentally mess up the order and potentially get really hard to debug issues.